.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
+}
```

Comment:
I don't think it's a good idea to include editor/ide configurations. Could you remove this file?